.github/workflows/smoke-test-daily.yml

```diff
@@ -44,36 +46,24 @@ jobs:
 
             - name: Run 'Update WooCommerce' test.
               working-directory: plugins/woocommerce
-              id: e2e-update
               env:
                   UPDATE_WC: true
-              run: pnpm exec playwright test --config=tests/e2e-pw/playwright.config.js update-woocommerce.spec.js
+              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js update-woocommerce.spec.js
```

Comment:
I added a separate Playwright config file for the daily tests so that it's easier to disable the shopper tests for the meantime